src/index.html

```diff
@@ -1,13 +1,289 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <link
+      rel="icon"
+      href="../src/images/icons/icon-page.png"
+      sizes="228x24"
+    >
+    <title>B&amp;O</title>
+    <link href="https://fonts.googleapis.com/css2?family=Manrope:wght@400;700&family=Roboto:wght@400;500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Landing Page</h1>
+  <body class="page__body">
+    <header class="header">
+      <div class="top-bar">
+        <a href="#" class="top-bar__logo">
+          <img src="./images/logo.png" alt="Bang&Olufsen logo">
+        </a>
+
+        <div class="top-bar__icons">
+          <a
+            href="tel:+1 234 5555-55-55"
```

Comment:
Remove spaces and dashes from the href attribute to avoid mistakes while redirecting